GPRegistryPolicyParser.psm1

```diff
@@ -184,7 +184,14 @@ Function Parse-PolFile
     $index = 0
 
     [string] $policyContents = Get-Content $Path -Raw
-    [byte[]] $policyContentInBytes = Get-Content $Path -Raw -Encoding Byte
+    if ( $PSVersionTable.PSVersion.Major -eq "5")
```

Comment:
> -eq [](start = 40, length = 4)

would it be -le instead?

Reply:
I don't understand what you want sorry ? I don't touch anything rather than the Verb an a switch to make it compatible with Powershell 7

Reply:
Yes your write it would be -le instead -eq.